### PR TITLE
Automate BZ 1384673

### PR DESCRIPTION
**Pull request by svtkachenko**
https://bugzilla.redhat.com/show_bug.cgi?id=1384673

Test is skipped because of BZ 1394390 but it is possible to run it with workaround, so test results with passed test:
```python
% pytest -v tests/foreman/ui/test_packages.py -k 'check_custom'
===================================================================== test session starts =====================================================================
platform linux2 -- Python 2.7.12, pytest-3.0.4, py-1.4.31, pluggy-0.4.0 -- /home/qui/code/venv/2/bin/python2
cachedir: .cache
rootdir: /home/qui/code/robottelo, inifile: 
plugins: xdist-1.15.0, html-1.12.0, cov-2.4.0
collected 6 items 

tests/foreman/ui/test_packages.py::PackagesTestCase::test_positive_check_custom_package_details PASSED

===================================================================== 5 tests deselected ======================================================================
========================================================== 1 passed, 5 deselected in 132.16 seconds ===========================================================
```
and skipped:
```python
% pytest -v tests/foreman/ui/test_packages.py -k 'check_custom'                         [git][robottelo/.][add_t_package]
===================================================================== test session starts =====================================================================

tests/foreman/ui/test_packages.py::PackagesTestCase::test_positive_check_custom_package_details <- robottelo/decorators/__init__.py SKIPPED

===================================================================== 5 tests deselected ======================================================================
========================================================== 1 skipped, 5 deselected in 95.56 seconds ===========================================================
```

### CODE REVIEW

File: tests/foreman/ui/test_packages.py [hunk 1]
```diff
@@ -132,6 +136,34 @@ def test_positive_check_package_details(self):
...
+        and then open it. Check that package details are available
+
+        @id: 679622a7-003e-4887-8622-b95b9468da7d
+
+        @Assert: Package is present inside of repository and it possible to
+        view its details
+
+        @CaseLevel: Integration
+
+        @BZ: 1384673
```
Comment (oshtaier) [inline tests/foreman/ui/test_packages.py]: `@BZ: 1384673, 1394390`

File: tests/foreman/ui/test_packages.py [hunk 2]
```diff
@@ -132,6 +136,34 @@ def test_positive_check_package_details(self):
                 ]
             )
 
+    @tier2
+    @skip_if_bug_open('bugzilla', 1394390)
+    def test_positive_check_custom_package_details(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_packages.py]: Do you go through `Manage packages` interface to check `1394390`

**Comment by oshtaier** [comment]
ACK

File: tests/foreman/ui/test_packages.py (hunk 2 above)
Comment (svtkachenko) [inline tests/foreman/ui/test_packages.py]: This test covers `1384673` that is related to `Content -> Packages` when uploading custom packages, while `1394390` is a blocker as it does not allow to upload custom packages and `Manage packages` related to `Content -> Repositories`. There is separate test for uploading  custom packages in repository test-suite, so there is no need to cover that in this test.

**Pull request merged**